### ISSUE TITLE
Ability to Configure UID/GID of nextcloud user

### ISSUE THREAD

**Issue by a575606** [nextcloud/docker#659]
I know this has been mentioned before, but if linuxserver can do it in their docker image, I don't understand why it's not possible to configure the UID and GID of the docker user that runs nextcloud. Having the file ownership set to user id 33 which doesn't exist makes permission management issues difficult to diagnose. For example, I can set cron jobs in synology dsm for users that don't exist. And I can't add access permission to the parent directory the nextcloud data resides in for a user which doesn't exist. I have lots of other docker images and nextcloud is literally the only one which doesn't support this feature, so I really hope you'll consider adding it.

**Comment by SnowMB** [comment]
Duplicate of #359

**Comment by a575606** [comment]
Correction: Reposted duplicate of #359 due to unsatisfactory conclusion reached when closing #359. Do I need to keep reposting until you take it seriously and leave open what is clearly a needed feature?

**Comment by SnowMB** [comment]
#359 is still open. It will be closed when somebody steps forward and implements it. Please do not open duplicates of open issues deliberately, that add nothing new to the issue.

The nextcloud docker image is a purely community driven project. If you are in desperate need of a feature you can either implement it yourself or find someone you can pay to implement it for you.

In either way it would be very much appreciated if you would share your additions with a pull request. 

In the mean time simply demanding that something should be implemented brings you nowhere.

**Comment by tilosp** [comment]
btw it should™️ work by using the --user flag to start the container

**Comment by a575606** [comment]
@snowMB I hear your point. The only thing I was demanding was some basic politeness. I don't think anyone appreciates having their issue closed as a duplicate when they refer to the previous issue in their post. I had just spent 4 or 5 hours trying to recover nextcloud from a minor update gone wrong, to the point of migrating everything to the linuxserver nextcloud image to try and complete the upgrade on a fresh install. Going through that process, I felt it worth making the case again for the the ability to set uid and gid. I didn't realize how much extra headache it caused until I had a problem with my install. 

But, you are quite right, it's not your problem I was having a bad day. And sorry for the way I came off. If this isn't a priority for you guys, I understand. Just thought it might be an easy fix since linuxserver implemented it in their image. 

Thanks tilosp, I'll look into the user runtime command. Have you tried ? Didn't see any documentation on changing user, and wondered if it might create problems
